configure

```diff
@@ -4197,6 +4208,25 @@ fi
 define_in_config_if_yes "$_vorbis" 'USE_VORBIS'
 echo "$_vorbis"
 
+#
```

Comment:
Shouldn't this check be moved down? It doesn't quite make sense to me to put it between the Vorbis and Tremor checks.

Reply:
Makes sense. I didn't really think about where I am placing the check. I will move it down.